tools/travis_script.sh

```diff
@@ -55,6 +49,14 @@ pip list
 section_end "Install.optional.dependencies"
 
 
+section "Build.docs"
+if [[ ($PY != 2.6) && ($PY != 3.2) ]]; then
+    export SPHINXCACHE=$HOME/.cache/sphinx
```

Comment:
I tried putting these on the same line and it worked.  Isn't bash fun?

https://travis-ci.org/blink1073/scikit-image/jobs/79185050